tests/run-internalbench.py

```diff
@@ -95,6 +151,10 @@ def main():
             continue
         test_dict[m.group(1)].append([t, None])
 
+    if pyb is not None:
+        print("Uploading bench harness.")
+        pyb.fs_put('internal_bench/bench_hw.py', 'bench.py')
```

Comment:
This is a reasonable way to do it, copy the file.  But it does require a writable filesystem on the target, for the current directory to be in the path, and it will be a bit slow due to filesystem writing.

An alternative way is to prepend the harness code to the script before sending it to the target.  Something like this should work:
```py
class bench_class:
    def run(test):
        ...
import sys
sys.modules['bench'] = bench_class
< here goes the actual benchmark test >
```